orion/evaluation/point.py

```diff
@@ -0,0 +1,180 @@
+""" The evaluator module provides a mechanic to assess the perfomance of
```

Comment:
I think that this docstring is outdated.

Reply:
Removed.